app/controllers/static_controller.rb

```diff
@@ -103,4 +111,60 @@ def default(input_data, default)
     end
   end
 
+  def seed_data_check()
+    if session[:is_match_data_complete] == nil || !session[:is_match_data_complete]
```

Comment:
I believe these checks are equivalent, you should be able to just use one of them